### PR TITLE
Further Edits to Mac Catalyst Post

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: _posts/2020-05-30-mac-catalyst-crash-hunt.md [hunk 1]
```diff
@@ -52,25 +52,25 @@ The crashers all circle around `NSTextCheckingController`, `NSBridgedTextCorrect
...
+In the first crash, we see `-[NSTextInputContext(NSTextInputContext_RemoteTextInput_UIKitOnMac) selectedRange_RTI]` calling `-[RTIDocumentState selectedTextRange]`. Let’s find the former. From [my earlier experiments with Marzipan](https://pspdfkit.com/blog/2018/porting-ios-apps-to-mac-marzipan-iosmac-uikit-appkit/), I know that a lot of UIKit-AppKit glue code is in UIKitMacHelper, in the iOSSupport system `/System/iOSSupport/System/Library/PrivateFrameworks/UIKitMacHelper.framework`, which is really a symlink to `/System/Library/PrivateFrameworks/UIKitMacHelper.framework`. `Versions/A/UIKitMacHelper` However, there’s no `selectedRange_RTI` implementation. Where else could it be?
```
Comment (deutschbitte) [inline _posts/2020-05-30-mac-catalyst-crash-hunt.md:69]: What does "Versions/A/UIKitMacHelper`" belong to? It's just standing alone right now.

"in the iOSSupport system `/System/iOSSupport/System/Library/PrivateFrameworks/UIKitMacHelper.framework`, which is really a symlink to `/System/Library/PrivateFrameworks/UIKitMacHelper.framework`. `Versions/A/UIKitMacHelper` However, there's no `selectedRange_RTI` implementation. Where else could it be?"

File: _posts/2020-05-30-mac-catalyst-crash-hunt.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ in NSObject:
 	isa (Class): __NSTextInputContextAuxiliaryStorage (isa, 0x1dffff8820fafd)
 ```
 
-The decompiled code calls `rdi = *([self auxiliary] + 0x58);` so we know it's a direct ivar access; we can also access it at runtime via KVC: 
+The decompiled code calls `rdi = *([self auxiliary] + 0x58);` so we know it’s a direct ivar access; we can also access it at runtime via KVC:
```
Comment (deutschbitte) [inline _posts/2020-05-30-mac-catalyst-crash-hunt.md:163]: What is "it" referring to?

"The decompiled code calls `rdi = *([self auxiliary] + 0x58);` so we know it's a direct ivar access; we can also access it at runtime via KVC:"

File: _posts/2020-05-30-mac-catalyst-crash-hunt.md (hunk 1 above)
Comment (steipete) [reply _posts/2020-05-30-mac-catalyst-crash-hunt.md:69]: This is part of the path, just broken into two pieces for better readability.

File: _posts/2020-05-30-mac-catalyst-crash-hunt.md (hunk 2 above)
Comment (steipete) [reply _posts/2020-05-30-mac-catalyst-crash-hunt.md:163]: it = this assembly command

**Pull request merged**